source/OutfitterPanel.cpp

```diff
@@ -438,7 +450,8 @@ void OutfitterPanel::Buy(bool alreadyOwned)
 			else
 			{
 				// Check if the outfit is for sale or in stock so that we can actually buy it.
-				if(!outfitter.Has(selectedOutfit) && player.Stock(selectedOutfit) <= 0)
+				if(!planet->Outfitter().Has(selectedOutfit) || Imports(customSales, selectedOutfit) ||
```

Comment:
This does seem like `import` means you can't buy the outfit, which shouldn't be the case.

Reply:
Wdym shouldnt be the case its the definition of import to not let you buy